dune-project

```diff
@@ -0,0 +1,3 @@
+(lang dune 1.11)
+(name ocaml-fileutils)
+(explicit_js_mode)
```

Comment:
Is this (explicit_js_mode) needed?

Reply:
If it's removed, then when you do `dune build @all` it fails if you don't have `js_of_ocaml-compiler`.